### ISSUE TITLE
Some Maps cant be played in Singleplayer

### ISSUE THREAD

**Issue by Mottemo** [speige/WC3MapDeprotector#28]
Hey,
some maps cant be played in singleplayer. they will detect if you are alone with a bot and wont do anything.
does the automatic file name detection work if it starts the map in singleplayer first and than i quit to the main screen and open the same map in LAN?

**Comment by speige** [comment]
Good point. I haven't actually tested this scenario, but I believe your approach should work. Let me know if it's not working for you.

I need to do more testing of "Live Game Scanner" because it seems like it isn't finding as many files as it should.
